lib/tbot/config/template_kubernetes.go

```diff
@@ -216,14 +224,6 @@ func (t *templateKubernetes) render(
 		// We only support directory mode for this since the exec plugin needs
 		// to know the path to read the credentials from, and this is
 		// unpredictable with other types of destination.
```

Comment:
Move this comment block near the moved code above?

Reply:
👍